### ISSUE TITLE
Training: learning rate experiments

### ISSUE THREAD

**Issue by inakbk** [mateuszmatu/OceanAI#191]
Currently running on:
`/scratch/project_465002266/experiments/learning_rate/`

two of them got nans in training, others we are waiting on.
